pkg/telemetry/collector.go

```diff
@@ -82,8 +82,13 @@ func (l *LeaderRunnable) Start(ctx context.Context) error {
 type snapshotCollector func(ctx context.Context, memberClusterMap map[string]ConfigClient, operatorClusterMgr manager.Manager, operatorUUID, mongodbImage, databaseNonStaticImage string) []Event
 
 // RunTelemetry lists the specified CRDs and sends them as events to Segment
-func RunTelemetry(ctx context.Context, mongodbImage, databaseNonStaticImage, namespace string, operatorClusterMgr manager.Manager, clusterMap map[string]cluster.Cluster, atlasClient *Client, configuredOperatorEnv util.OperatorEnvironment) {
-	Logger.Debug("sending telemetry!")
+func RunTelemetry(leaderTrace context.Context, mongodbImage, databaseNonStaticImage, namespace string, operatorClusterMgr manager.Manager, clusterMap map[string]cluster.Cluster, atlasClient *Client, configuredOperatorEnv util.OperatorEnvironment) {
+	Logger.Debug("Collecting telemetry!")
```

Comment:
uppercased Logger is quite surprising to me. I know it's been there before this change, but can you explain why it's a different convention than other log *zap.SugaredLog conventions we use?

Reply:
The reason was to expose a global package level Logger which is always ready and correctly initialized for the independent telemetry module. This saves us the need for creating a new sugaredLogger everytime + the potential of configuring it wrongly